content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -382,6 +382,81 @@ You can also see the `retainKeys` strategy in the [OpenApi spec](https://raw.git
 And you can see the `retainKeys` strategy in the
 [Kubernetes API documentation](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#deploymentspec-v1-apps).
 
+## Use subresource flag with kubectl patch command to update the deployment
```

Comment:
I would move this inside:
```markdown
## Alternate forms of the kubectl patch command
```

Also, a text suggestion:

```suggestion
## Update an object's replica count using `kubectl patch` with `--subresource` {#scale-kubectl-patch}

{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
```
(is this right, alpha in v1.24?)

Reply:
Yes, it is alpha in v1.24.
Ref: https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/2590-kubectl-subresource/kep.yaml#L24

Reply:
addressed